sbi/utils/user_input_checks_utils.py

```diff
@@ -368,7 +368,7 @@ def support(self):
         # Wrap as `independent` in order to have the correct shape of the
         # `log_abs_det`, i.e. summed over the parameter dimensions.
         return constraints.independent(
-            constraints.cat(supports, dim=1, lengths=self.dims_per_dist),
+            constraints.cat(supports, dim=-1, lengths=self.dims_per_dist),
```

Comment:
please add it to the PR description such that we can later add it to the `CHANGELOG.md`